content/hi/client_server_architecture.md

```diff
@@ -0,0 +1,17 @@
+---
+title: क्लाइंट-सर्वर आर्किटेक्चर
```

Comment:
```suggestion
title: क्लाइंट-सर्वर आर्किटेक्चर (Client-Server Architecture)
```

Reply:
We're adding English version in brackets for title on all the terms as a standard. It helps to find the definitions easily and also helps us appear on the search results.

Reply:
I'm committing this suggestion to move this PR forward